Level0/Voting/index.html

```diff
@@ -1,41 +1,44 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title>Trying to make the frontend to send the data</title>
+	<title>Voting</title>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/web3/1.2.7-rc.0/web3.min.js"></script>
 	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
 	<script type="text/javascript" src="./index.js"></script>
 	<link rel="stylesheet" type="text/css" href="./index.css">
 </head>
+
 <body background="1.webp">
-	<div id ="do">
-	<h2 id= "top">SIMPLE VOTING </h2>
-</div>
-<div>
-	<img id = "web3"src="web31.gif">
+	<div id="do">
+		<h2 id="top">VOTING</h2>
+	</div>
+	<div>
+		<img id="web3" src="web31.gif">
 
-	<img id = "web5"src="./web5.png">
-</div>
-	<button id ="btn">
+		<img id="web5" src="./web5.png">
+	</div>
+	<button id="btn1">
 		VOTE
 	</button>
-	<button id ="btn1">
+	<button id="btn2">
 		VOTE
 	</button>
 
-	<div><button id ="res">
-		RESULT
-	</button></div>
+	<div><button id="btn3">
+			RESULT
+		</button></div>
 
 	<div>
-		<button id = "show-vote1">
-			TOTAL VOTE FOR CANDIDATE 1
+		<button id="show-vote1">
+			Total vote for candidate 1
 		</button>
-		<button id = "show-vote2">
-			TOTAL VOTE FOR CANDIDATE 2
+		<button id="show-vote2">
+			Total vote for candidate 2
 		</button>
 	</div>
 </body>
+
```

Comment:
Do  one thing  increase    the font size of the  TOTAL VOTE FOR CANDIDATE 1 and TOTAL VOTE FOR CANDIDATE 2.
 and also check if this responsive or not ,and if not please fix this .
@ankur0904 Hope u understand .